scripts/update_json_docs.sh

```diff
@@ -23,25 +29,33 @@ function pushDocs () {
     echo "Nothing to commit."
   fi
   cd ..
+
+  echo "Done deploying JSON documentation."
 }
 
 function cleanSubmodule () {
-    echo "Cleaning up..."
+    echo "Cleaning up!"
     git submodule deinit -f ghpages
     git reset HEAD .gitmodules
     git reset HEAD ghpages
     rm -rf ghpages
     rm -f .gitmodules
     rm -rf .git/modules/ghpages
+    echo "Done cleaning up!"
 }
 
+# Run this to verifiy that the docs build successfully.
+buildDocs $TRAVIS_BRANCH
+
 if [ "${TRAVIS_BRANCH}" == "master" ] && [ "${TRAVIS_PULL_REQUEST}" == "false" ]; then
+    buildDocs $TRAVIS_BRANCH
```

Comment:
It will have been called. I'll take it out here and just use the results from how it ran before.